nemo_text_processing/text_normalization/hy/data/currency.tsv

```diff
@@ -0,0 +1,52 @@
+aed	դիրհամ
+ARS	արգենտինական պեսո
+֏	դրամ
+֏	հայկական դրամ
+a$	ավստրալական դոլար
+br	բելառուսական ռուբլի
+₿	բիթքոին
+£	ֆունտ
+£	բրիտանական ֆունտ
+ca$	կանադական դոլար
+元	չինական յեն
+元	յեն
+kr	դանիական կրոն
+$	դոլար
+€	եվրո
+₾	վրացական լարի
+₾	լարի
+₹	հնդկական ռուփի
+₹	ռուփի
+﷼	պարսկական ռիալ
+﷼	ռիալ
+₪	իսրայելական շեկել
+₪	շեկել
+¥	ճապոնական յեն
+¥	յեն
+₸	ղազախական տենգե
+₸	տենգե
+som	ղրղզական սոմ
+som	սոմ
+ل.ل	լիբանանյան ֆունտ
+ل.ل	լիբանանյան լիրա
+nz$	նորզելանդական դոլլր
+kr	նորվեգական կրոն
+zł	լեհական զլոտի
+zł	զլոտի
+£	ֆունտ ստերլինգ
+£	ֆունտ
+₽	ռուսական ռուբլի
+₽	ռուբլի
+rsd	սերբական դինար
+s$	սինգապուրի դոլար
+₩	կորեական վոն
+kr	շվեդական կրոն
+chf	շվեյցարական ֆրանկ
+£s	սիրիական ֆունտ
+₺	թուրքական լիրա
+₴	ուկրաինական գրիվնա
+$	ամերիկյան դոլար
```

Comment:
If you ran `normalize.py` on just --text="**$123**" you will get **23** instead of "**հարյուր քսաներեք ամերիկյան դոլար**"

Reply:
Works fine on my side, the thing is in Armenian most of the time people write currency after cardinal or decimal not before. So in this case I get output *$ հարյուր քսաներեք*. But if you try *123$* it works perfectly fine and outputs  *հարյուր քսաներեք դոլար*, the other names for the same currencies are added for future pull requests where will be added non-deterministic behavior.